### PR TITLE
dep: upgrade help.client

### PR DESCRIPTION
**Pull request by Blackbaud-ColbyWhite**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -507,9 +507,9 @@
       "dev": true
     },
     "@blackbaud/help-client": {
-      "version": "2.0.2",
-      "resolved": "https://registry.npmjs.org/@blackbaud/help-client/-/help-client-2.0.2.tgz",
-      "integrity": "sha512-C0JezDxjm6zGcS5jyD6xXHxQ8xMxMva6ztDkKJv9pFs+NmR9xQ3evn+Mpl+yBjKAXCJFMDXJVudk6gE466osuw==",
+      "version": "2.1.0",
+      "resolved": "https://blackbaud.pkgs.visualstudio.com/_packaging/Blackbaud-npm/npm/registry/@blackbaud/help-client/-/help-client-2.1.0.tgz",
```
Comment (Blackbaud-ColbyWhite) [inline package-lock.json]: it's not clear to me that this is a good idea. it seems not to be. but i also dont know how to make my system support both.

Comment (Blackbaud-BobbyEarl) [reply package-lock.json]: Agreed it's not a good idea.  Let's see if we can work together to resolve it offline.

**Comment by Blackbaud-BobbyEarl** [comment]
Build is failing for an unrelated reason.  I'll work on a PR to address that.

**Comment by Blackbaud-BobbyEarl** [comment]
I believe https://github.com/blackbaud/skyux-sdk-builder/pull/159 addresses why this PR is failing in Travis.

**Pull request merged**